GenderGapsInCollegeDegrees.md

```diff
@@ -0,0 +1,160 @@
+
+## Clean up the chart
+1. Let's explore the gender gap in STEM fields, such as science, technology, engineering, and mathematics.
```

Comment:
Also, it's good to link what you want to explore (the gender gap) with the dataset.  How will the dataset help you explore the gap? What information does it contain?